### PR TITLE
Migration paths

### PR DESCRIPTION
**Pull request by oxidand**
Allow to pass multiple paths to where tenant migrations are stored. This way if the project have many migrations and they are separated into modules we can include them through config, e.g. `config(['tenancy.migrations_paths' => ['/path/one', '/path/two']]);`

### CODE REVIEW

File: src/Traits/DealsWithMigrations.php [hunk 1]
```diff
@@ -12,6 +12,6 @@ protected function getMigrationPaths()
             return parent::getMigrationPaths();
         }
 
-        return [config('tenancy.migrations_directory', database_path('migrations/tenant'))];
+        return config('tenancy.migrations_paths', [database_path('migrations/tenant')]);
```
Comment (stancl) [inline src/Traits/DealsWithMigrations.php]: Fallback to previous config to maintain backwards compatibility:

```suggestion
        return config('tenancy.migrations_paths', [config('tenancy.migrations_directory') ?? database_path('migrations/tenant')]);
```

File: tests/CommandsTest.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ public function setUp(): void
     {
         parent::setUp();
 
-        config(['tenancy.migrations_directory' => database_path('../migrations')]);
+        config('tenancy.migrations_paths', [database_path('migrations/tenant')]);
```
Comment (stancl) [inline tests/CommandsTest.php]: Why the change of the path?

Comment (oxidand) [reply tests/CommandsTest.php]: didn't think about, my mistake

Comment (stancl) [reply tests/CommandsTest.php]: Please commit this:
```suggestion
        config('tenancy.migrations_paths', [database_path('../migrations')]);
```

File: src/Traits/DealsWithMigrations.php (hunk 1 above)
Comment (stancl) [reply src/Traits/DealsWithMigrations.php]: Please commit this suggestion.

Comment (stancl) [reply src/Traits/DealsWithMigrations.php]: When you commit this, I can merge the pull request.

File: src/Traits/DealsWithMigrations.php [hunk 3]
```diff
@@ -12,6 +12,6 @@ protected function getMigrationPaths()
             return parent::getMigrationPaths();
         }
 
-        return [config('tenancy.migrations_directory', database_path('migrations/tenant'))];
+        return config('tenancy.migrations_paths', [config('tenancy.migrations_directory') ?? database_path('migrations/tenant')]);
```
Comment (stancl) [inline src/Traits/DealsWithMigrations.php]:
```suggestion
        return config('tenancy.migration_paths', [config('tenancy.migrations_directory') ?? database_path('migrations/tenant')]);
```

File: tests/CommandsTest.php [hunk 4]
```diff
@@ -18,7 +18,7 @@ public function setUp(): void
     {
         parent::setUp();
 
-        config(['tenancy.migrations_directory' => database_path('../migrations')]);
+        config('tenancy.migrations_paths', [database_path('../migrations')]);
```
Comment (stancl) [inline tests/CommandsTest.php]: We need to use `config([])` because we are setting config values.

```suggestion
        config(['tenancy.migration_paths', [database_path('../migrations')]]);
```

File: tests/TestCase.php [hunk 5]
```diff
@@ -96,7 +96,7 @@ protected function getEnvironmentSetUp($app)
             'tenancy.redis.tenancy' => env('TENANCY_TEST_REDIS_TENANCY', true),
             'database.redis.client' => env('TENANCY_TEST_REDIS_CLIENT', 'phpredis'),
             'tenancy.redis.prefixed_connections' => ['default'],
-            'tenancy.migrations_directory' => database_path('../migrations'),
+            'tenancy.migrations_paths' => [database_path('../migrations')],
```
Comment (stancl) [inline tests/TestCase.php]:
```suggestion
            'tenancy.migration_paths' => [database_path('../migrations')],
```

**Review by stancl — CHANGES_REQUESTED** [review]
Sorry, final round of changes. Since `paths` is plural, `migration` should be singular. Also fixed one bug.

Please click Commit suggestion for each suggestion and I'll merge the PR.

File: src/Traits/DealsWithMigrations.php (hunk 3 above)
Comment (stancl) [reply src/Traits/DealsWithMigrations.php]: @curious-jim Please commit this suggestion too.

File: tests/TestCase.php (hunk 5 above)
Comment (stancl) [reply tests/TestCase.php]: @curious-jim Please commit this suggestion too.
